### ISSUE TITLE
could not determine kind of name for C.rocksdb_backup_engine_open_opts

### ISSUE THREAD

**Issue by lishunan246** [linxGnu/grocksdb#85]
I got these errors when running go build.

```
# github.com/linxGnu/grocksdb
../../go/pkg/mod/github.com/linx!gnu/grocksdb@v1.7.7/backup.go:44:13: could not determine kind of name for C.rocksdb_backup_engine_open_opts
../../go/pkg/mod/github.com/linx!gnu/grocksdb@v1.7.7/backup.go:136:2: could not determine kind of name for C.rocksdb_backup_engine_restore_db_from_backup
```

I am using Ubuntu 22.04 LTS, librocksdb-dev (6.11.4-3) from apt and grocksdb@v1.7.7.

Any help would be appreciated!

**Comment by linxGnu** [comment]
Hello @lishunan246 
It looks to me that you are using `grocksdb v1.7.7` which is compatible to latest rocksdb (v7.5.3)

However, your rocksdb (installed via api) is v6.11.4. From 6.11.4 to 7.x, there are many APIs changes (newly added, some are removed). That's why you see error when building

To fix this, please use `grocksdb v1.6.22`: https://github.com/linxGnu/grocksdb/releases/tag/v1.6.22

**Comment by lishunan246** [comment]
Thanks! 
v1.6.22 works for me
